### PR TITLE
Convert click to argparse for flexibility

### PR DESCRIPTION
**Pull request by illusional**
I couldn't make `@click` do what we wanted, eg: parsing only a subset of `sys.argv`, but script isn't a named parameter, it's just the remainder of args. So I've reverted this back to argparse (and implemented a cheap "confirm" function).

### CODE REVIEW

File: analysis_runner/cli.py [hunk 1]
```diff
@@ -153,7 +132,64 @@ def _get_google_auth_token() -> str:
...
+    parser.add_argument('script', nargs=argparse.REMAINDER, default=[])
+
+    return parser.parse_args(args)
+
+
 if __name__ == '__main__':
-    # Disable pylint because click decorates the function in a specific way
-    # pylint: disable=no-value-for-parameter
-    main()
+    main_from_args()
```
Comment (lgruen) [inline analysis_runner/cli.py:195]: Shouldn't this be `main_from_args(sys.argv)` or similar? Where do the `args` come from? :)

Comment (illusional) [reply analysis_runner/cli.py:195]: Argparse automatically pulls from `sys.argv`.

Comment (lgruen) [reply analysis_runner/cli.py:195]: Got it, thanks!

Comment (illusional) [reply analysis_runner/cli.py:195]: I added a comment to clarify this in code.

**Comment by illusional** [comment]
@vladsaveliev tests are passing.

**Review by vladsaveliev — APPROVED** [review]
Forgot to approve :)

**Pull request merged**